addons/web/static/src/views/fields/char/char_field.xml

```diff
@@ -10,13 +10,14 @@
                 class="o_input"
                 t-att-class="{
                     'o_field_translate': isTranslatable,
-                    'o_field_placeholder': hasDynamicPlaceholder
+                    'o_field_placeholder': hasDynamicPlaceholder,
+                    'text_muted': !this.props.record.data[props.name],
```

Comment:
Create a new css custom class for that ?

Reply:
this should not be done in web, it's related to the other part of the feature

Reply:
Also, the placeholder_field feature should be tested (in char_field.test.js)

Reply:
Yeah that was one thing i wasn't sure about, if we let that in account entirely or part in web, i will do that thanks !